modules/nw-network-flows-object.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/tracking-network-flows.adoc
+
+[id="nw-network-flows-object_{context}"]
+= Network object configuration for tracking network flows
+
+The following YAML shows the configuration fields for network flows collectors in the Cluster Network Operator configuration.
+
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  exportNetworkFlows: <.>
+    netFlow:
+      collectors:
+        - <netflow-collector-1>:2056
+        - <netflow-collector-2>:2056
```

Comment:
There doesn't seem to be a standard port for NetFlow, `2056` seems to be the convention and is the default for the example goflow tool.

https://github.com/cloudflare/goflow

Reply:
Thanks for all the port information.  I failed to consider that it might not be accurate. Eek.